source/locales/nl.json

```diff
@@ -163,7 +163,7 @@
   "DNS Provider": "DNS-provider",
   "dns-notes": "<p>Google Public DNS logt je ISP en locatiegegevens permanent voor analyse. Bovendien wordt je IP-adres voor 24 uur opgeslagen.</p><p><strong>OpenNIC</strong> heeft geen officiële policy aangaande log query privacy / anonimisatie. Meer informatie in het <a href='https://wiki.opennic.org/opennic:tier2'>OpenNIC-wiki</a>.</p>",
   "Anonymizing Networks": "Anonieme netwerken",
-  "Tor Hidden Services": "Tor-verborgen Services",
+  "Tor onion services": "Tor-verborgen Services",
```

Comment:
Are these keys expected somewhere else in the codebase? I haven't checked at _all_ but it seems like this might wreak havoc

Reply:
Also the translations are presumably wrong too, we should probably get rid of them

Reply:
I'm not sure, while Tor hidden service is a deprecated name for this technology, I think it's better to use it than have no translation at all.